tools/build_rules/closure/closure_js_binary.bzl

```diff
@@ -57,7 +74,7 @@ def _impl(ctx):
       "--closure_entry_point=%s" % ctx.attr.main,
       "--js_output_file=%s" % ctx.outputs.out.path,
       "--language_in=ECMASCRIPT5_STRICT",
-      "--manage_closure_dependencies",
+      "--dependency_mode=LOOSE",
```

Comment:
Are you sure this works? Doesn't setting the `--dependency_mode` flag mean you're also required to use `--entry_point=goog:foo.main` instead of `--closure_entry_point=foo.main`?